### DISCUSSION TITLE
can maxFileSize in config be changed after a while?

### DISCUSSION THREAD

**Discussion by AliEmili** [keystonejs/keystone#9195] [Questions] [answered]
I have set a fixed quantity for the maxFileSize property in config, now as my users increased, I need more space to store uploaded data, can maxFileSize be changed in config without any consequences and side-effects?

**Comment by dcousens** [comment, marked as answer]
At least for now, the `config.server.maxFileSize` configuration variable is only used by the `graphql-upload` express middleware.  This currently only prevents particular uploads at runtime, and has no downstream effects.

https://github.com/keystonejs/keystone/blob/554a10a8ed514d6c0c1d3045957140c033baa64d/packages/core/src/lib/createExpressServer.ts#L104-L106
